src/Exception/MissingApiConfigException.php

```diff
@@ -11,17 +11,11 @@
 namespace BitBag\ShopwareInPostPlugin\Exception;
 
 use Shopware\Core\Framework\ShopwareHttpException;
-use Symfony\Component\HttpFoundation\Response;
 
-final class ApiDataException extends ShopwareHttpException
+final class MissingApiConfigException extends ShopwareHttpException
```

Comment:
Why did you remove the status codes? This is actually pretty nifty here as otherwise the default implementation is:

```php
    public function getStatusCode(): int
    {
        return Response::HTTP_INTERNAL_SERVER_ERROR;
    }
```

Reply:
What about the other exception?